package.json

```diff
@@ -59,7 +61,8 @@
   },
   "peerDependencies": {
     "react": ">=16.8.6",
-    "react-native": ">=0.60.0"
+    "react-native": ">=0.60.0",
+    "react-native-windows": ">=0.62"
```

Comment:
should this be in the module always? I forget where we landed on this @jonthysell @acoates-ms